src/Overlay/Modal/Modal.tsx

```diff
@@ -10,11 +10,20 @@ type ModalProps = {
     hideCancelButton: boolean;
     isOpen: boolean;
     onClose?: () => void;
+    classNames: ClassNames;
+  };
+
+  type ClassNames = {
+    header?: string;
+    body?: string;
+    footer?: string;
+    cancelBtn?: string;
```

Comment:
Could we please use `button` instead of `btn` please.

Reply:
Sure, done.